clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -193,6 +195,8 @@ const SelectInput = ({
       isClearable={isClearable}
       instanceId={`select-${field.name}`}
       isMulti={isMulti}
+      isDisabled={isDisabled}
+      menuPosition="fixed"
```

Comment:
Allows the menu to be in a fixed position. Allows menu to be visible.

![Screenshot 2023-02-16 at 11 53 10 AM](https://user-images.githubusercontent.com/68459950/219433710-a1283bf7-aab4-455f-91fc-c86b0c7d5cf6.png)

Reply:
I'm giving this some testing because settings this for all of our uses of select could have unexpected layout effects. What was the problem before? Was it getting cut off by the modal edge?

Reply:
Yeah, so this is how the dataset editor works without fixed:
![good-no-fixed](https://user-images.githubusercontent.com/2236777/219465321-c166d780-99fe-40c4-aa1b-c47a2d707a6a.gif)

And this is what happens with the `menuPosition="fixed"`
![fun-fixed](https://user-images.githubusercontent.com/2236777/219465326-226582d0-a4d2-4c3e-8b3c-7cddfd26fca5.gif)
(With wider browser window the menu is completely invisible)

Fixed position is a classic monster.

Reply:
Posible solutions:
1. Add min/max menu height as props that get [passed along to react select](https://react-select.com/props#:~:text=Set%20the%20maximum%20height%20of%20the%20menu.). Then we can avoid the menu extending beyond the modal (or causing scroll) entirely.
2. Add `menuPosition` as a prop so that it can be used here, but is not set to `fixed` by default.

Reply:
Issue has been resolved.